Cargo.toml

```diff
@@ -21,7 +21,7 @@ env_logger = "0.9.0"
 clap = "3.0.8"
 hyper = { version = "0.14.16", features = ["full"] }
 signal = {path = "examples/signal" }
-tokio-util = "0.6.9"
+tokio-util = {version="0.6.9", features = ["full"] }
```

Comment:
Why was this change required?

Reply:
to fix this error: 

error[E0432]: unresolved import `tokio_util::codec`
  --> examples/ice-restart/ice-restart.rs:11:17
   |
11 | use tokio_util::codec::{BytesCodec, FramedRead};
   |                 ^^^^^ could not find `codec` in `tokio_util`

Reply:
my rational in this PR was: I m gonna fix all the errors and the warnings related to the example I'm modifing. 
Do you agree with that?

Reply:
while I was waiting, I've also prepared a branch where I've fixed all the warnings in this project: https://github.com/jurgob/examples/tree/fix_warnings. 

I would do a PR with that after this one is merged. 

(in this PR I've also tested manually my changes, in that branch I did not, but the change is quite the same in all the branch). 


If you prefer I can merge that branch in this PR so there's gonna be 0 warnings

let me know @k0nserv

Reply:
anyway, I see your concern there. 
I've changed that importing only the codec feature

Reply:
I've limited the warning fix to this example, but I've fixed that error to make the pass the ci checks

Reply:
Hmm I just noticed that it was already broken on `main` which means CI failing for this PR is probably a red herring anyway

Reply:
so how do you suggest I proceed @k0nserv ?

in the meantime I've fixed the CI problem in another PR: https://github.com/webrtc-rs/examples/pull/14

the problem is that PR is including this PR (which is really 1 readme line change and 3 lines of code). 

can we close this and we just move on the next pr?

Reply:
as an alternative, I can remove the example change that I did from the pr #14 , and once that is merged we can continue doing this PR.